cluster/gce/gci/mounter/Makefile

```diff
@@ -0,0 +1,27 @@
+# Copyright 2016 The Kubernetes Authors.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+TAG?=v1
+REGISTRY?=gcr.io/google_containers
+IMAGE=gci-mounter
+
+all: container
+
+container:
+	docker build --pull -t ${REGISTRY}/${IMAGE}:${TAG} .
```

Comment:
Other images do the build as a two step process, to ensure `latest` is set correctly:
```MAKE
	docker build --pull -t ${REGISTRY}/${IMAGE} .  # Build new image and automatically tag it as latest
	docker tag ${REGISTRY}/${IMAGE} ${REGISTRY}/${IMAGE}:${TAG} # Add the version tag to the latest image
```